source/OutfitterPanel.cpp

```diff
@@ -72,6 +79,39 @@ namespace {
 		}
 		return toRefill;
 	}
+
+
+
+	bool IsLicense(const string &name)
+	{
+		return name.ends_with(LICENSE);
+	}
+
+
+
+	string LicenseRoot(const string &name)
+	{
+		return name.substr(0, name.length() - LICENSE.length());
+	}
+
+
+
```

Comment:
Namespace functions typically only have one blank line between them.
```suggestion

	bool IsLicense(const string &name)
	{
		return name.ends_with(LICENSE);
	}

	string LicenseRoot(const string &name)
	{
		return name.substr(0, name.length() - LICENSE.length());
	}

```